documentation/docs/guide/content/Block-Patterns.md

```diff
@@ -0,0 +1,58 @@
+# Patterns
+
+WordPress block patterns are a collection of pre-made design elements that you can use for creating custom content layouts faster. These patterns include items like pre-built multi-column layouts, media and text patterns, call-to-action patterns, headers, buttons, and more.
+
+<img src="/bcgov-wordpress-block-theme/add-block-icon.png" align="left" height="50" hspace="15">
+
+To use block patterns, you need to edit the WordPress post or page where you want to use the block pattern. On the page/post edit screen, click on the Add Block button to open the block inserter. From there, switch to the Patterns tab to view available block patterns.
+
+You can also view BCGov specific block patterns in different categories like general, headers, footers, pages, and more including any site specific patterns developed specifically for sites such as CleanBC.
+
+## Understanding Patterns
+
+- Patterns are combinations of core block components and custom classes set up in very specific configurations.
+- Patterns are typically designed to be discreet page specific sections and are not meant to be combined within other patterns.
+- Patterns are usually contained in a Group block at its top most level when using the List View.
+- Pre-defined styling can be changed by moving or altering pattern structure while others may cause design failures.
+- Be very cautious moving patterns into other patterns.
+
+## BCGov Block Patterns
+
+Patterns are separated into different categories such as General, Banners, Header/Footer, Page Layout, Post Queries, etc. These categories are used to separate intended purpose and also keep the list of patterns from getting too large and inhibit browsing. Should you know the name of a pattern you can use the search tool in the Block Inserter to filter and find the one you need.
+
+Many patterns are available for use as part of the default BCGov base theme. They offer layouts such as Alternating Cards, Card with Image Overlay, Card with Under Image Portrait 2-Up, Cards Portrait 3-up, Detail Card with Icons, Hero Banner, Long Card, Quote, Small Quote with Image, and many more.
+
+## Creating Custom Patterns
+
+<img src="/bcgov-wordpress-block-theme/patterns-admin-menu.png" align="right" height="175" hspace="15">
+
+## Enabling printing of an image
```

Comment:
Actually now that I look at these headings, they don't fit what is underneath them so I guess they should just be deleted.